docs/cookbook/customizing_form_builder_email_notifications.md

```diff
@@ -0,0 +1,24 @@
+# Customizing Form Builder email notifications
+
+!!! enterprise
+    
+    ## Override email template  
+    
+    Email is one of the Submit button options you can choose from in the Form Builder Interface.
+    It allows you to gather readers' email addresses to send notifications to.
+    To customize Form Builder email submission notification you need to override Twig template `form_builder/form_submit_notification_email.html.twig`.
+    It is built with two blocks, subject and body. Each of them is rendered independently and consists of three sets of parameters.
+    
+    |Parameter|Type|Description|
+    |---------|----|-----------|
+    |`content`|`eZ\Publish\API\Repository\Values\Content\Content`|Name of the form, its Content Type|
+    |`form`|`EzSystems\EzPlatformFormBuilder\FieldType\Model\Form`|Definition of the form|
+    |`data`|`EzSystems\EzPlatformFormBuilder\FieldType\Model\FormSubmission`|Sent data|  
+    
+    By adjusting them to your needs, you will change your email template.
+    
+    ## Configure sender details
```

Comment:
IMHO this should be moved to https://doc.ezplatform.com/en/2.3/guide/configuration or to similar page dedicated to the Form Builder. Without this configuration the delivery of notification e-mails will be not possible.

Reply:
I added the information to page dedicated to Form Builder. However I will also leave the same info in the cookbook, as recipes are complete tutorials and we want to avoid adding links to other pages in the documentation for additional steps.